apps/site/lib/site_web/templates/layout/app.html.eex

```diff
@@ -70,6 +70,14 @@
         <%= content_tag :main, render(@view_module, @view_template, assigns), id: "main", tabindex: -1 %>
       </div> <%# /container %>
       <%= render __MODULE__, "_footer.html", conn: @conn %>
+
+      <%# Show a warning when browser is Internet Explorer %>
+      <%= if Application.get_env(:site, :dev_server?) do %>
+        <script defer src="<%= "#{Application.get_env(:site, :webpack_path)}/iewarning.js" %>"></script>
+      <% else %>
+        <script defer src="<%= static_url(@conn, "/js/react.js") %>"></script>
```

Comment:
Is this script tag needed if React is already imported in the IEWarning?

Reply:
Now that I think about it it'd be nice if we didn't have to load React for everyone who visits 😕